### ISSUE TITLE
SUggestion

### ISSUE THREAD

**Issue by mjcedwards** [DougManuel/CO2-logger#5]
Hi - Interesting project.  One suggestion I would make is that if you used a WEMOS microcontroller - i.e. a WiFi enabled Arduino then you could get the time from NTP, and could report results in real time using BLYNK - https://blynk.io/

I set up a temperature and humidity monitor this way, and it will alert me if the temperature exceeds a certain set point.

Good Luck....Mark

**Comment by DougManuel** [comment]
WiFi-enabled! I've been wanting to get into microcontrollers and blynk.io looks great. Consider it done!

**Comment by GalaxyKitten101** [comment]
Great idea!

**Comment by mjcedwards** [comment]
Yes - Blynk is pretty easy to use and free as long as you keep things simple.  If you are also thinking of doing some Android Apps, check out the MIT App Inventor - also very easy to use.
http://appinventor.mit.edu/
